.github/workflows/test.yml

```diff
@@ -2,11 +2,21 @@ on: push
 name: Run acceptance tests
 
 jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
++ to Node 18, maybe we can add an nvmrc as well?

Reply:
As an FYI, [Node.js v20](https://nodejs.org/en/blog/announcements/v20-release-announce) was just released today 🚀 The "Call to action!" section at the end of that page recommends that we use either v18 or v20:

> Also of note is that Node.js 14 will go End-of-Life in April 2023, so we advise you to start planning to upgrade to Node.js 18 (LTS) or Node.js 20 (soon to be LTS).

> Please, consider that Node.js 16 (LTS) will go End-of-Life in September 2023, which was brought forward from April 2024 to coincide with the end of support of OpenSSL 1.1.1. You can read more details about that decision at https://nodejs.org/en/blog/announcements/nodejs16-eol/.

Reply:
Is anything stopping us from using v20 right ahead?

Reply:
The reason I used 16 is because the action itself uses v16. In the `action.yml` file we say the following:
```yml
runs:
  using: 'node16'
```

I'm up for updating this as well, but I couldn't find anywhere the available strings that mention the supported Node versions when creating a Node action.

I suppose changing it to `node18` or `node20` should do the trick, but I'm not sure. 🤔 

Regardless, what I want to say is that if we decide to switch it here, we should switch it everywhere.

Reply:
Agreed, IMO we can close this thread now. As noted [here](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-using-nodejs-v16) (and elsewhere as we searched for examples), `node16` is currently the recommended version for actions.